src/components/SearchComponents/Search.page.tsx

```diff
@@ -90,16 +123,15 @@ const Search: FC<SearchProps> = (props) => {
     searchRank: number;
   }) => {
     if (searchHit) {
-      const $ = cheerio.load(searchHit.title);
       track.searchResultOpened({
         keyStageSlug: searchHit.keyStageSlug || "",
         keyStageTitle: searchHit.keyStageTitle as KeyStageTitleValueType,
         subjectTitle: searchHit.subjectTitle,
         subjectSlug: searchHit.subjectSlug,
         unitName:
-          searchHit.type === "unit"
-            ? $.text()
-            : convertUnitSlugToTitle(searchHit.buttonLinkProps.unitSlug), // unit name without highlighting html tags,
+          searchHit.type === "lesson"
+            ? searchHit.unitTitle.replace(/(<([^>]+)>)/gi, "")
+            : searchHit.title.replace(/(<([^>]+)>)/gi, ""), // unit name without highlighting html tags,
```

Comment:
since this regex is used in 3 places that I can see it's worth pulling it out into something with a name that conveys what it does (then we don't need the comments either)